plugins/search-backend-node/src/types.ts

```diff
@@ -51,3 +57,20 @@ export interface RegisterDecoratorParameters {
    */
   types?: string[];
 }
+
+/**
+ * A type of function responsible for translating an abstract search query into
+ * a concrete query relevant to a particular search engine.
+ */
+export type QueryTranslator = (query: SearchQuery) => any;
```

Comment:
```suggestion
export type QueryTranslator = (query: SearchQuery) => unknown;
```

it's usually best to avoid using `any` unless necessary; `unknown` is the stricter, better `any` in a lot of cases